<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QWidget" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>630</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Должности/Исключения</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="position_work_edit"/>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="delete_cause">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
           </property>
           <property name="text">
            <string>Удалить причину</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="delete_position">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
           </property>
           <property name="text">
            <string>Удалить должность</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QTableView" name="view_position">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="add_position">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
           </property>
           <property name="text">
            <string>Добавить должность</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="add_cause">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
           </property>
           <property name="text">
            <string>Добавить причину</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QTableView" name="view_cause">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QLineEdit" name="cauese_excude_edit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Категории</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_15">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_28">
         <item>
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="2">
            <widget class="QGroupBox" name="groupBox_16">
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Квалификация</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_31">
              <item>
               <widget class="QRadioButton" name="kval_dan_2">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Дан</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="kval_ku_2">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Кю</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="save_template_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>55</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{

background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
             </property>
             <property name="text">
              <string>Сохранить шаблон</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QGroupBox" name="groupBox_15">
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Степень квалификации</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_32">
              <item>
               <widget class="QLineEdit" name="kval_from_2">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="placeholderText">
                 <string>  От</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="kval_to_2">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="placeholderText">
                 <string>  До(включительно)</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_14">
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Возраст</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_29">
              <item>
               <widget class="QLineEdit" name="age_from_2">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="placeholderText">
                 <string>  От</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="age_to_2">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="placeholderText">
                 <string>  До (включительно)</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QGroupBox" name="groupBox_12">
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Пол</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_30">
              <item>
               <widget class="QRadioButton" name="gender_M_2">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>М</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="gender_J_2">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Ж</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QPushButton" name="create_category_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>55</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{

background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
             </property>
             <property name="text">
              <string>Создать категорию</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="delete_category_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>55</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{

background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
             </property>
             <property name="text">
              <string>Удалить категорию</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QPushButton" name="delete_all_category_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>55</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{

background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
             </property>
             <property name="text">
              <string>Удалить все категории</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="load_template_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>55</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{

background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
             </property>
             <property name="text">
              <string>Загрузить шаблон</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="view_all_categorys_2">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Шрифт</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>351</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>Размер шрифта</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSpinBox" name="FontSize_SpinBox"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
